src/main/scala/org/globalnames/parser/ParserRelaxed.scala

```diff
@@ -5,53 +5,45 @@ import org.parboiled2.CharPredicate.{Digit, Alpha, LowerAlpha}
 
 class ParserRelaxed extends ParserClean {
```

Comment:
we don't, it will be gone, but before it is gone we need to decide on a few things